### ISSUE TITLE
Question about training settings when submitting to MOT test server 

### ISSUE THREAD

**Issue by sanghoooon** [SysCV/qdtrack#122]
Dear authors, 
I appreciate you for making the implementation publicly available. 
I am a newbie in multiple object tracking and conducting research based on your approach.

I am trying to submit my results on MOT test server, and curious about your training configuration for submitting to the official MOT challenge server.
For example:
* the ratio between training/validation sets
* the total number of epochs and/or learning rate scheduling, if they differ from the provided config file.

I am asking since it seems normal to use more than half of the training samples (as defined in `half-val_cocoformat.json` for MOT dataset), especially when submitting results to a test server in order to maximize the performance.

Thanks in advance.
